client/src/components/TOTP/Verify.js

```diff
@@ -127,7 +127,7 @@ class Verify extends Component {
 
     return (
       <a href={method.supportLink} target="_blank" rel="noopener noreferrer">
-        { i18n._t('TOTPVerify.HOW_TO_USE', 'How to use authenticator app.') }
+        { i18n._t('TOTPVerify.HOW_TO_USE', 'How to use authenticator apps.') }
```

Comment:
Can you use `method.supportText` so it uses the backend value? You could leave this as a fallback as you've done in MFA if it's undefined

Reply:
@NightJar can you please update this? The other two PRs are merged now

Reply:
Sorry, I made the change and pushed... but forgot to commit first 😓